sql/plsql/PlSqlLexer.g4

```diff
@@ -2326,7 +2326,9 @@ CARRET_OPERATOR_PART:      '^';
 TILDE_OPERATOR_PART:       '~';
 EXCLAMATION_OPERATOR_PART: '!';
 GREATER_THAN_OP:           '>';
+GREATER_OR_EQ_THAN_OP:     '>' SPACE* '=';
```

Comment:
This doesn't look right. Why would there be spaces in an operator? Also, `GREATER_THAN_OR_EQ_OP` would be a more correct name.

Reply:
Page 3-5 of Database PL/SQL Language Reference, v21c: "Do not embed any [other] characters (including whitespace characters) inside a delimiter."